reboot/celeryconfig.py

```diff
@@ -10,8 +10,7 @@
 worker_prefetch_multiplier = 1
 worker_concurrency = 10
 accept_content = ['json', 'pickle']
-result_backend = broker_url
-# result_backend = 'django-db'
+result_backend = config("CELERY_RESULT_BACKEND", default=broker_url)
```

Comment:
slightly worried about the compatibility of broker_url and result_backend being different protocol. i guess we can test in staging. Is the goal to keep it as amqp on prod and rpc on dev?